src/Core/CustomMethods.php

```diff
@@ -279,10 +279,12 @@ protected function removeMethodsFrom($property, $index = null)
         $methods = $this->findMethodsFromExtension($extension);
         if ($methods) {
             foreach ($methods as $method) {
-                $methodInfo = self::$extra_methods[$class][$method];
+                if (isset(self::$extra_methods[$class][$method])) {
```

Comment:
How about doing an early `continue` instead of indenting the contents of this statement?